tests/bgp/test_docker_restart.py

```diff
@@ -0,0 +1,58 @@
+'''
+
+This script is to Verify BGP facts are not altered when docker container restarts.
+
+Step 1: Verify BGP is up and gather basic facts
+Step 2: Restart docker container for BGP service
+Step 3: Verify BGP facts are same as before restart
+
+'''
+import logging
+import pytest
+import time
+from tests.common.utilities import wait_until
+
+logger = logging.getLogger(__name__)
+bgp_sleep = 30
+
+pytestmark = [
+    pytest.mark.topology('t2')
```

Comment:
Why is this specific to T2 topology? I suppose that this test applies to pizzabox switches as well.

Reply:
I mark tests to the topology that I have tested them on and I only have a T2 testbed.  So it is possible it would work fine on other topologies, but since I don't know for sure they get marked as T2 only.